recipes/aurelia-cli/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set version = "3.0.3" %}
+
+package:
+  name: aurelia-cli
+  version: {{ version }}
+
+source:
+  url: https://registry.npmjs.org/aurelia-cli/-/aurelia-cli-{{ version }}.tgz
+  sha256: 7f92172432c1fd5ab58ef2abf8fa6b477f8bce8efed8a2a84a7e94a38b92af06
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - make
+    - nodejs
+    - pnpm
+    - pnpm-licenses
```

If there is compiled code, nodejs should also be in build
```suggestion
    - pnpm-licenses
  host:
    - nodejs
```